data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+				`"OK, I will see what I can find."`
+					goto find
+				`"No, I don't want to get involved in this."`
+					goto nothelp
+			`	She considers her thoughts briefly, then chants, "How fast a ship can move through an area of deep space basically depends on how fast they can collect fuel. Ramscoops were common at the time of the exodus, and if humanity has developped better ramscoops or other means of generating fuel, these would significantly improve their speed at exploring or crossing large volumes of space."`
+			choice
+				`"OK, I will see what I can find."`
+				`"No, I don't want to get involved in this."`
+					goto nothelp
+			label find
+			`	Taely nods "I will look forward to seeing what you find. In the meantime, I should get back to work." With that, she swings over the railing and climbs quickly down the wall in the direction of the shipyard, seemingly unconcerned about the height.`
+			label nothelp
+			`	Taely looks at you with a blank expression, then chants in a monotone voice "If that is your choice, you are unlikely to find much of interest to do among us." With that, she vaults over the edge of the railing and drops from view. As you quickly peer over the railing, you discover that she had must have caught a handhold somehow, as she is rapidly moving down the wall in a controlled fashion with ease of a spider.`
+				decline
+	to complete
+		or
+			has "Remnant: Heavy Laser: done"
+			has "Remnant: Catalytic Ramscoop: done"
+			has "Remnant: Plasma Cannons: done"
+			has "Remnant: Electron Beam: done"
+			has "Remnant: D94-YV Shield Generator: done"
+			has "Remnant: S-970 Regenerator: done"
+
+
+
 mission "Remnant: Heavy Laser"
 	name "Retrieve Heavy Lasers for the Remnant"
-	description "A Remnant engineer on <planet> has offered to pay you <payment> in exchange for obtaining two heavy lasers for them to study."
+	description "A Remnant engineer has offered to pay you <payment> in return for delivering two heavy lasers to <planet> for them to study."
 	minor
-	source
+	source "Viminal"
+	destination
 		government "Remnant"
 	to offer
-		has "Remnant: Technology Available: offered"
-		not "Remnant: Catalytic Ramscoop: active"
-		not "Remnant: Catalytic Ramscoop: done"
-		not "Remnant: Plasma Cannons: active"
-		not "Remnant: Plasma Cannons: done"
-		random < 50
+		or
+			has "Remnant: Tech Retrieval: active"
+			has "Remnant: Tech Retrieval: done"
 	on offer
 		require "Heavy Laser"
 		conversation
-			`As you are browsing the spaceport shops, a woman approaches you and introduces herself as a researcher at one of the Remnant weapons facilities. She awkwardly pauses before beginning to recite a quick tune: "We can see that your ship has beam weapons that match human techniques, but are unlike anything we have record of. What kind of weapons are they?"`
+			`You remember that Taely was interested in seeing examples of technology from human space. Would you like to show her your heavy laser cannons?`
```

Comment:
Cannons?

Reply:
Removed "Cannons," since the weapon doesn't actually mention the word anywhere. 
I included that word as a way of differentiating it from the heavy laser turret, which won't work for this mission.